aten/src/ATen/functorch/BatchRulesLinearAlgebra.cpp

```diff
@@ -280,6 +280,62 @@ static void expect_at_least_rank(
       rank, " dimensions instead.");
 }
 
+fourOutputs _scaled_dot_product_efficient_attention_batch_rule(
+  const Tensor& query, optional<int64_t> query_bdim,
+  const Tensor& key, optional<int64_t> key_bdim,
+  const Tensor& value, optional<int64_t> value_bdim,
+  bool compute_log_sumexp,
+  double dropout_p,
+  bool is_causal,
+  c10::optional<double> scale
+) {
+  const auto query_ = query_bdim.has_value() ? reshape_dim_into(*query_bdim, 0, query) : query;
+  const auto key_ = key_bdim.has_value() ? reshape_dim_into(*key_bdim, 0, key) : key;
+  const auto value_ = value_bdim ? reshape_dim_into(*value_bdim, 0, value) : value;
+
+  const auto res = at::_scaled_dot_product_efficient_attention(query_, key_,
+      value_, compute_log_sumexp, dropout_p, is_causal, scale);
+  const auto batch_size = query.size(*query_bdim);
+  const auto res0_ = reshape_dim_outof(0, batch_size, std::get<0>(res));
+  const auto res1_ = reshape_dim_outof(0, batch_size, std::get<1>(res));
+  const auto res2_ = reshape_dim_outof(0, batch_size, std::get<2>(res));
+  const auto res3_ = reshape_dim_outof(0, batch_size, std::get<2>(res));
```

Comment:
Looks like a typo.  This is the same as res2.

Reply:
oops thanks!

Reply:
shoudl be fixed in new commit